tests/testthat/test-modelAR.R

```diff
@@ -87,8 +86,14 @@ if (require(testthat)) {
     expect_true(length(forecast(winennet, h = 2, xreg = matrix(2, 2, 3))$mean) == 2L)
     ## Test if h matches xreg
     expect_true(length(forecast(winennet, h = 5, xreg = matrix(2, 2, 3))$mean) == 2L)
-    expect_warning(forecast(winennet2, xreg = matrix(2, 2, 3))$mean, "different column names") %>%
-      expect_equal(forecast(winennet, xreg = matrix(2, 2, 3))$mean)
+    expect_warning(
+      expect_equal(
```

Comment:
previously, `expect_warning(x, ...)` would return `x`, but now it returns the `warning` condition, so we have to reverse the order here & a few other places